cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -68,6 +69,24 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
       });
     }
+
+    it('should render a visible placeholder image within a media block', () => {
+      cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+        ({ body }) => {
+          // `video` blocks can also contain audio, so this test
+          // checks if both media types have a placeholder image.
```

Comment:
How does it check both?

Reply:
It checks both, because audio _also_ comes through in a video block.
I've updated the comment, so it doesn't read as if we're doing anything special here.

Reply:
It checks both, because audio also comes through in a video block.
I've updated the comment, so it doesn't read as if we're doing anything special here.